proxy/envoy/resources.go

```diff
@@ -88,9 +88,9 @@ const (
 	RedisProxyFilter = "redis_proxy"
 
 	// RedisDefaultOpTimeout is the op timeout used for Redis Proxy filter
-	// Currently it is set to 30ms (conversion happens in the filter)
+	// Currently it is set to 30ms
 	// TODO - Allow this to be configured.
-	RedisDefaultOpTimeout = 30 * time.Second
+	RedisDefaultOpTimeout = 30 * time.Millisecond
```

Comment:
seems unrelated ? isn't this short ?

Reply:
completely unrelated, just accidentally got swept into the commit

Reply:
should not effect anything.